### ISSUE TITLE
`integers` and `randint` raise TypeError if rng is not provided

### ISSUE THREAD

**Issue by ricardoV94** [pymc-devs/pytensor#97]
We should correct these methods to create the right type of RNG if they are not provided by the user (that's what `super().make_node()`) would do anyway

https://github.com/pymc-devs/pytensor/blob/237f54f9a673022c0c0f02a7322b55ddc6a949fc/pytensor/tensor/random/basic.py#L1914-L1919

https://github.com/pymc-devs/pytensor/blob/237f54f9a673022c0c0f02a7322b55ddc6a949fc/pytensor/tensor/random/basic.py#L1964-L1970

**Comment by Dhruvanshu-Joshi** [comment]
is this issue still relevant after #691 ?

**Comment by ricardoV94** [comment]
> is this issue still relevant after #691 ?

No, good catch. It's no longer relevant
